pages/linux/ip-route.md

```diff
@@ -0,0 +1,32 @@
+# ip route
+
+> IP Routing table management subcommand.
+> More information: <https://www.man7.org/linux/man-pages/man8/ip-route.8.html>.
+
+- Display the routing table:
+
+`ip route show/list`
```

Comment:
```suggestion
`ip route show`
```
I think we should only use `show` here. Both work and do exactly the same, but `list` isn't in the man page.

Reply:
or `ip route {{show|list}}` ?

Reply:
Is there a difference between `show` and `list`?

Reply:
If they're identical, then hmmm, I'm not sure what to do here.  Probably what @Waples suggests; that seems like the best idea.